.changeset/beige-readers-pull.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
should be removed

Reply:
removed it.
I've added it to see if that was the issue to generate a snaptshot :/